openslides/motions/static/js/motions/base.js

```diff
@@ -201,6 +325,24 @@ angular.module('OpenSlidesApp.motions', [
                 getSearchResultSubtitle: function () {
                     return "Motion";
                 },
+                /**
```

Comment:
Here we have the docstring just before the function which is ok for me. But some lines later the docstring is inside. It should be consistent. So you can move the docstring inside the function or open an issue to cleanup all other functions.